lib/rules/jsx-no-hardcoded-content.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
     const dom = defaultOptions.dom || {};
     const optionsForAllDOM = dom['*'] || defaultDOMOptions['*'];
 
+    // eslint-disable-next-line prefer-object-spread
```

Comment:
Are you sure the version of node this runs in doesn't have spread?

Reply:
We've broken other shopify apps in the past for things like this so i'm playing it safe here.